packages/query/src/common/alex-sdk/alex-sdk.hooks.ts

```diff
@@ -1,7 +1,6 @@
 import { useCallback } from 'react';
 
-import { Currency, type TokenInfo } from 'alex-sdk';
-import { AlexSDK } from 'alex-sdk';
+import { AlexSDK, Currency, type TokenInfo } from 'alex-sdk';
```

Comment:
Why wasn't this double import a lint error 🤔

Reply:
`no-duplicate-imports` was missing from the mono repo eslint-config pkg, so adding it in this PR.

Reply:
<img width="521" alt="Screenshot 2024-07-03 at 8 29 22 AM" src="https://github.com/leather-io/mono/assets/6493321/a28383f1-4556-4e8c-9f1a-94660f51a789">